SConstruct

```diff
@@ -0,0 +1,138 @@
+"""
+Copyright (c) 2012-2014 Stanford University
+Copyright (c) 2018 Henry Corrigan-Gibbs
+
+Permission to use, copy, modify, and distribute this software for any 
```

Comment:
When adding to Firefox you should check that the license is ok to use.

Reply:
@rhelmer Rob: Let me know if you are going to need to include this file in the browser code. If using the ISC license here will be a problem, I can throw out this SConstruct file and write a new MPL-licensed one from scratch.